src/components/Header/index.jsx

```diff
@@ -16,9 +17,14 @@ export default function Header() {
 
     return (
 
-        <header className="bg-slate-50 p-4">
-            <h1 className="pb-3 text-3xl font-semibold">Invoices</h1>
-            <p className="pb-3 text-slate-500" >There are {unpaid} unpaid invoices</p>
+        <header className="bg-slate-50 pt-4 pb-2 mb-2 flex justify-around border-b border-gray-400">
+            <div>
+                <h1 className="pb-3 text-3xl font-semibold">Invoices</h1>
```

Comment:
Can you make this a link back home? Just gives the user another way to get back other than using cancel invoice